### ISSUE TITLE
And again (after patch ) Timeout on map server iRO Renewal

### ISSUE THREAD

**Issue by shacalif** [OpenKore/openkore#1324]
Same behavior, we need to update some crypt keys may be.

![image](https://user-images.githubusercontent.com/32203940/32087206-bc83cbe0-bab0-11e7-8b17-f0a5dc359d68.png)

**Comment by sctnightcore** [comment]
download new openkore and test again ! .

**Comment by fadreus** [comment]
@shacalif and again, you should know allanon will update after the patch released.
and again, everyone expect he will sit in front of the PC waiting eagerly to update for the sake of the world like some hero in comic as if he has nothing else better to do with his life anymore in this world..
and again, still not bother reading other issues.
and again, even after countless time this scenario happened, human never learn from the past.
and again, why can't you wait?
and again, why can't you check?
and again, and again, and again..

**Comment by shacalif** [comment]
I'm not complaint about the work of the people here, actually I'm more than happy.
Sorry if the post did that impression... my complaints are only with warpportal just that... And of course I can wait as always do., 
:-)

**Comment by fadreus** [comment]
Ah, no no. I didn't say you were complaining. it just a bit of joke, bad habit of mine. xD
I'm just giving some insight how to solve your issue since the problem solved 3 hours ago.

The bug report templates clearly mention what to do before starting new issue.
You missed those 2; having latest version **_(unless you came from the past using time machine from 3 hours ago)_** and didn't check existed issue ( #1323  & #1321 ). You would't starting new issue if you already did that, don't you?

![image](https://user-images.githubusercontent.com/32188494/32088427-c941d466-bb15-11e7-83bd-c18c91cbb64f.png)

Well, don't forget to close your issue after you done.

<details>
<summary>Hahaha..</summary>

Since most people only come here for answer and after get what they want, they simply never look it back here again; until they have other issue.

Just some minor problem for dev (I think) need to check every issues and then need to close it.
Help the community by making it more organized.

</details>

**Comment by shacalif** [comment]
Yeah my bad, i forgot to check for any changes in the master branch before to open the issue.

Thanks, is working now!!
